src/compiler/types.ts

```diff
@@ -6774,6 +6762,7 @@ export type TypeMapper =
     | { kind: TypeMapKind.Function; func: (t: Type) => Type; debugInfo?: () => string; }
     | { kind: TypeMapKind.Composite | TypeMapKind.Merged; mapper1: TypeMapper; mapper2: TypeMapper; };
 
+/** @internal */
 // dprint-ignore
 export const enum InferencePriority {

```

Comment:
This is used by nobody: https://github.com/search?q=%2F%5Cbts%5C.InferencePriority%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code